packages/ckeditor5-image/src/image/insertimagecommand.ts

```diff
@@ -18,7 +18,7 @@ import type ImageUtils from '../imageutils.js';
  * and it is also available via aliased `imageInsert` name.
  *
  * In order to insert an image at the current selection position
- * (according to the {@link module:widget/utils~findOptimalInsertionRange} algorithm),
+ * (according to the {@link module:engine/model/schema~Schema#findOptimalInsertionRange} algorithm),
```

Comment:
This was using the widget implementation of this method. I think this should not change.

Reply:
Adressed in 24ac1fa